gcc/rust/parse/rust-parse-impl.h

```diff
@@ -1385,6 +1387,25 @@ Parser<ManagedTokenSource>::parse_vis_item (AST::AttrVec outer_attrs)
 	  lexer.skip_token (1); // TODO: is this right thing to do?
 	  return nullptr;
 	}
+    case ASYNC:
```

Comment:
I feel like we could have a new function here, something along `parse_async_item`, we already have `parse_const_item`.

Reply:
Understood, I'll make this change.

Reply:
Hey @P-E-P , in our parser we have a function called `parse_async_block`, which has just has a defination and no caller in the entire codebase, would you like me to make a new `parse_async_item` function or just ammend my code to `parse_async_block`

Reply:
I think `parse_async_block` was made exclusively for async block (`async {}`) you should probably create a new function instead of using this one.

Reply:
Oh ok, got it :)